apps/builder/app/shared/style-object-model.ts

```diff
@@ -0,0 +1,155 @@
+import { properties } from "@webstudio-is/css-data";
+import { StyleValue } from "@webstudio-is/css-engine";
+import type { Instance, StyleDecl, StyleSource } from "@webstudio-is/sdk";
+
+/**
+ *
+ * use cases where computing is called
+ *
+ * 1. compiler
+ *
+ * put the whole tree into the model once
+ * lazily compute each point in the tree reusing cached cascaded values
```

Comment:
```suggestion
 * Put the whole tree into the model once,
 * lazily compute each point in the tree reusing cached cascaded values.
```

Reply:
etc, try to make sentences complete, its easier to read this way

Reply:
those are not sentences but machine instructions like pseudo code